CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## _Unreleased_
+- Add `--exclude-missing` when running `auto-pr status`
+  - Allows you to remove the `Missing PRs` section from the output
```

Comment:
Can you change this to 1.0.7 so we can release it immediately after merging?

Reply:
👍